src/components/ImageView/index.native.js

```diff
@@ -64,22 +64,25 @@ class ImageView extends PureComponent {
         ImageSize.getSize(this.props.url).then(({width, height}) => {
             let imageWidth = width;
             let imageHeight = height;
-            const aspectRatio = (imageHeight / imageWidth);
-
-            // Fit the image to windowWidth
-            imageWidth = Math.round(this.props.windowWidth);
-            imageHeight = Math.round(imageWidth * aspectRatio);
+            const containerWidth = Math.round(this.props.windowWidth);
+            const containerHeight = Math.round(this.props.windowHeight - variables.contentHeaderHeight);
+
+            const aspectRatio = Math.min(containerHeight / imageHeight, containerWidth / imageWidth);
+
+            // resize image to fit the screen dimensions if it small than screen else resize the smaller dimension only to avoid the resize issue on android
+            if (imageHeight < containerHeight && imageWidth < containerWidth) {
+                imageHeight *= aspectRatio;
+                imageWidth *= aspectRatio;
+            } else if (imageHeight > imageWidth) {
+                imageHeight *= aspectRatio;
+            } else {
```

Comment:
move this to a new line please

Reply:
why move this to a new line? 
it will give lint issue

Reply:
@rushatgabhane why move this to a new line? 
it will give lint issue

Reply:
TIL we have this lint rule. Thanks @ahmdshrif